yarn.lock

```diff
@@ -9610,6 +9673,14 @@ reason-react@0.8.0:
   resolved "https://registry.yarnpkg.com/reason-react/-/reason-react-0.8.0.tgz#a55143f95a872596c92b54595e7904c03ee7c9db"
   integrity sha512-97AfK3eCF6vfP8rnbq3k6GgXsZt//OiPMINTUk5luJ8W0Wt/JaYsDG6eWKP8Zx3gPpxSU1EPjB1pUm6o6fBnFA==
 
+reason-react@^0.5.3:
```

Comment:
Bah, any way to use 0.8? I know we rely on gatsby-plugin-reason but maybe updating gatsby-plugin-reason to support 0.8?

Reply:
Good call!